src/ert/_c_wrappers/job_queue/workflow_job.py

```diff
@@ -147,9 +147,7 @@ def run(self, ert: "EnKFMain", arguments: List[Any]) -> Any:
             self.__script = ExternalErtScript(ert, self.executable)
         else:
             raise UserWarning("Unknown script type!")
-        result = self.__script.initializeAndRun(
-            self.argumentTypes(), arguments
-        )
+        result = self.__script.initializeAndRun(self.argumentTypes(), arguments)
```

Comment:
Seems like some difference in black makes this line change from one commit to the next. Could you run black on both commits so that the diff is smaller?